visual-recognition/src/main/java/com/ibm/watson/visual_recognition/v4/VisualRecognition.java

```diff
@@ -199,8 +198,6 @@ public VisualRecognition(String versionDate, String serviceName, Authenticator a
    * @return a {@link ServiceCall} with a response type of {@link Collection}
    */
   public ServiceCall<Collection> createCollection(CreateCollectionOptions createCollectionOptions) {
-    com.ibm.cloud.sdk.core.util.Validator.notNull(
```

Comment:
Why? I think we need createCollectionOption to not be null

Reply:
fixed in [0f6d07af0fa398a550ed2df90eff4f06f1f46c93](https://github.com/watson-developer-cloud/java-sdk/pull/1128/commits/0f6d07af0fa398a550ed2df90eff4f06f1f46c93)